### ISSUE TITLE
how to keep Hs after docking

### ISSUE THREAD

**Issue by Lili-Cao** [gnina/gnina#270]
### Issue summary
Hi, 

I am wondering if it is possible to keep the protons of ligands after docking, is so, how should I do that?

Thanks!
Lili

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
Python version:

**Comment by dkoes** [comment]
No, but is easy enough to add them back with your favorite cheminformatics toolkit.

**Comment by Lili-Cao** [comment]
but it is not easy to get the pronation exactly as the docked ones right?

Den mån 22 juli 2024 kl 14:43 skrev David Koes ***@***.***>:

**Comment by Lili-Cao** [comment]
but it is not easy to get the pronation exactly as the docked ones right?

**Comment by Lili-Cao** [comment]
Maybe not only the protonation state but also the geometry would be different?

**Comment by dkoes** [comment]
The point of docking is to generate a new geometry. If you have examples that you think are problematic please share them.

**Comment by Lili-Cao** [comment]
Hi,  the geometry of heavy atoms are ok from gnina but how about the protons?

**Comment by dkoes** [comment]
I find hydrogen atom geometry uninteresting.
